tests/deriver/disabled_test_performance.py

```
@@ -0,0 +1,295 @@
+
+            mock_get_dupes.return_value = []
+            mock_create_doc.return_value = None
+
+            start_time = time.time()
+
+            # Test processing in chunks
+            chunk_size = 20
+            for i in range(0, len(large_fact_list), chunk_size):
+                chunk = large_fact_list[i : i + chunk_size]
+                unique_facts = await store.remove_duplicates(chunk)
+                await store.save_facts(unique_facts)
```

Pylint (3.3.7)

[error] Instance of 'CollectionEmbeddingStore' has no 'save_facts' member

(E1101)